std_msgs/Header header
uint32 flag_state
